clang/include/clang/Driver/Options.td

```diff
@@ -931,7 +931,8 @@ def O : Joined<["-"], "O">, Group<O_Group>,
 def O_flag : Flag<["-"], "O">, Visibility<[ClangOption, CC1Option, FC1Option]>,
   Alias<O>, AliasArgs<["1"]>;
 def Ofast : Joined<["-"], "Ofast">, Group<O_Group>,
-  Visibility<[ClangOption, CC1Option, FlangOption]>;
+  Visibility<[ClangOption, CC1Option, FlangOption]>,
+  HelpText<"Deprecated; use -O3, possibly with -ffast-math">;
```

Comment:
```suggestion
  HelpText<"Deprecated; use '-O3', possibly with '-ffast-math'">;
```

Reply:
I changed the wording, so this is no longer applicable.

Reply:
(But I made sure to put arguments in quotes)